web/client/components/misc/enhancers/emptyState.jsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const {branch} = require('recompose');
+const DefaultEmptyComponent = require("../EmptyView");
+
+
+/**
+* Empty State enhancer. Enhances an object adding a (localizable) tooltip.
```

Comment:
I think this documentation is copied from tooltip.jsx

Reply:
good catch. Going to fix it